internal/controller/configuration/utils.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package configuration
+
+import (
+	"os"
+
+	"go.uber.org/zap"
+)
+
+func GetPodNamespace(log *zap.Logger) string {
```

Comment:
This is only used in `main` maybe it should just live there as a helper instead of in the `configuration` package?